### ISSUE TITLE
Run|Debug codelens bounce when input

### ISSUE THREAD

**Issue by tjx666** [rust-analyzer/rust-analyzer#11553]
OS: MacOS 12.2.2
VSCode: 1.65.0-insider
rust-analyzer: pre-release v0.3.952
rustc: 1.59.0

@alexdima Don't know whether it is VSCode editor issue.

code in video:

```rust
/*
结构体
 */

#[allow(unused)]

fn main() {
    // ------------------------ 定义结构体 -----------------------------
    // 注意是使用逗号分隔字段，习惯了在 ts 里面用分号分隔接口字段，切换到 rust 到容易敲错
    // aaaaaaaaaaaaaaa
    struct Layer {
        name: String,
        left: i32,
        top: i32,
        width: i32,
        height: i32,
    }
}
```

https://user-images.githubusercontent.com/41773861/155765638-edc35241-521d-4d65-871a-9bca04933559.mov

**Comment by Veykril** [comment]
This should be a vscode issue, we don't do anything in our client-code that affects the layout of these.

**Comment by alexdima** [comment]
@tjx666 We have seen this before at https://github.com/microsoft/vscode/issues/113384 but I could never reproduce to try and investigate it. Is this triggered only with some specific user settings?

**Comment by tjx666** [comment]
@alexdima

I finally found it iss related with:

```
"window.zoomLevel": 1,
```

If I remove this setting, the codelens won't bounce.

**Comment by tjx666** [comment]
Can you reproduce now? @alexdima

**Comment by alexdima** [comment]
:+1: I suggest to continue tracking at https://github.com/microsoft/vscode/issues/144612

**Comment by tjx666** [comment]
fixed in vscode
